archinstall/default_profiles/desktops/cosmic.py

```diff
@@ -8,7 +8,7 @@
 
 class CosmicProfile(XorgProfile):
 	def __init__(self):
-		super().__init__('cosmic-epoch', ProfileType.DesktopEnv, description='')
+		super().__init__('cosmic-epoch', ProfileType.DesktopEnv, description='', advanced=True)
```

Comment:
There was also a new greeter option added in archinstall/default_profiles/profile.py does that also need to be behind the flag?

Reply:
Ah good catch, I think that should be behind the flag too. I'll update the pr with this and a fix for the other comment.